twisted/topfiles/8597.feature

```diff
@@ -0,0 +1 @@
+Defined twisted.internet.interfaces.ITLSContextFactory which defines the interface provided both by the old-style twisted.internet.ssl.ContextFactory class and the newer twisted.interface.ssl.CertificateOptions class. This is a precursor to formally deprecating the former class in favour of the latter.
```

Comment:
(nitpick) the fqdn should always be first, so this would be "twisted.internet.interfaces.ITLSContextFactory has been added, which defines..."